Changelog

```diff
@@ -1,17 +1,57 @@
-Version 3.3, 2020-XX-XX
+Version 3.3, *2020-XX-XX*
+
+  Features:
+    * New token type: WebAuthn/FIDO2 token (#1468)
+      * WebAuthn hardware tokens can now be used with privacyIDEA
+    * New token type: Indexed Secret. (#1986)
+      * A challenge-response token that asks the user for positions from a list of secrets.
+    * *OPEN New sophisticated logging facility (#1580)*
+      * *TEXT TEXT TEXT*
+
+  Fixes:
+    * Fixed unauthorized statistics view (#1238)
```

Comment:
In the past we used "Fix". Please do so, do not use "Fixed" and do not use "Fixes".

Reply:
ok